build-tools/packages/build-cli/src/library/repoPolicyCheck/fluidBuildTasks.ts

```diff
@@ -249,20 +171,28 @@ function findTscScripts(json: Readonly<PackageJson>, project: string): string[]
 	addIfDefined(findScript(json, `tsc --project ${project}`));
 	addIfDefined(findFluidTscScript(json, project));
 	addIfDefined(findTscMultiScript(json, project));
-	return tscScripts.length > 0 ? tscScripts : undefined;
+	if (tscScripts.length === 1) {
+		return tscScripts[0];
+	}
+	if (tscScripts.length === 0) {
+		return undefined;
+	}
+	throw new Error(`${project} used in scripts ${tscScripts.join(", ")}`);
 }
 
 interface EslintConfig {
```

Comment:
Nit: have we looked to see if `@types/eslint` exposes types for this sort of thing?

Reply:
@types/eslint doesn't provide projects as that is plugin/parser specific. For our ushe it is from @typescript-eslint/eslint-plugin.
From that package set the type is available from @typescript-eslint/utils, but it requires Node16 resolution that is pending in PR #20972. It could be derived from constants in @typescript-eslint/eslint-plugin constants, but direct dep on that creates additional peer dependency that we don't want to maintain. I'll add notes.